packages/client/src/sync/fetcher/accountfetcher.ts

```diff
@@ -108,21 +114,24 @@ export class AccountFetcher extends Fetcher<JobTask, AccountData[], AccountData>
   /** The range to eventually, by default should be set at BigInt(2) ** BigInt(256) + BigInt(1) - first */
   count: bigint
 
-  /** Contains known bytecodes */
-  trie: Trie
-
   storageFetcher: StorageFetcher
 
   byteCodeFetcher: ByteCodeFetcher
 
+  trieNodeFetcher: TrieNodeFetcher
+
   accountTrie: Trie
 
   accountToStorageTrie: Map<String, Trie>
 
+  /** Contains known bytecodes */
```

Comment:
Shouldn't all these references be to trie nodes and not bytecode?  Isn't that specific to the `bytecodeFetcher`?

Reply:
I used a trie to keep all of the codes in for this first iteration. It's not a trie that needs healing.